src/dispatcher.js

```diff
@@ -86,22 +85,9 @@ var dispatcher = module.exports = dispatcher = {
 	* @private
 	*/
 	listen: function(inListener, inEventName, inHandler) {
-		if (inListener.addEventListener) {
-			this.listen = function(inListener, inEventName, inHandler) {
-				inListener.addEventListener(inEventName, inHandler || dispatch, false);
-			};
-		} else {
-			//enyo.log("IE8 COMPAT: using 'attachEvent'");
-			this.listen = function(inListener, inEvent, inHandler) {
-				inListener.attachEvent("on" + inEvent, function(e) {
-					e.target = e.srcElement;
-					if (!e.preventDefault) {
-						e.preventDefault = this.iePreventDefault;
-					}
-					return (inHandler || dispatch)(e);
-				});
-			};
-		}
+		this.listen = function(inListener, inEventName, inHandler) {
```

Comment:
There's no need for the reassignment logic any longer since we're not altering based on platform. We can just update the body of `listen()` to be
```
listen: function (inListener, inEventName, inHandler) {
	inListener.addEventListener(inEventName, inHandler || dispatch, false);
}
```